_capabilities/cognitivesalience.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Cognitive Models for Common Ground Modeling"
+excerpt: "We model both the AI and the human performer in a common modeling framework and use cognitive salience to reveal their respective mental models"
+tags: # Select from this set
+  - Autonomy
+  - Computer Vision
+  - Reinforcement Learning
+  - Human-Machine Teaming
+  - Saliency
+   
+submission_details:
+  resources: # List any resources associated with the contribution. Not all sections are required
+    papers:
+      - https://iccm-conference.neocities.org/2020/papers/Contribution_255_final.pdf
+      - https://baicsworkshop.github.io/pdf/BAICS_36.pdf
+      - https://iccm-conference.neocities.org/2019/proceedings/papers/ICCM2019_paper_53.pdf
+    demos:
+      - https://colab.research.google.com/drive/1tf5gYVab3GSpPsTUxp34PVxNCIbA19FC?usp=sharing
+   
+  authors:
+    - Sterling Somers (1)
+    - Constantinos Mitsopolous (1)
+    - Christien Lebiere (1)
+    - Peter Perolli (2)
+    - Joel Schooler (2)
+    - Robert Thomson (3)
+  organizations:
+    1. Department of Psychology, Carnegie Mellon University
+    2. Institute for Human and Machine Cognitive (IHMC)
+    3. Department of Psychology, United States Military Academy West Point
+  
+  point_of_contact:
+    name: Sterling Somers
+    email: sterling@sterlingsomers.com
+---
+   
+## Overview
+We demonstrate the use of cognitive architectures as a common framework for modeling AIs and human performers in a task. The mental model of each performer is revealed using cognitive salience, which is intended as a basis for explanation.
+
+At it's simplest you can use a memory model and instance-based learning in pyactup
```

Comment:
You may want to consider including the pyactup library in the software pointers, since you reference it here. You can try adding this in the resources section after the papers:

```
software:
    - https://halle.psy.cmu.edu/pyactup/
```

Reply:
If you do end up adding this software resource, please consider also updating the license field. You can link to a license like such (e.g. using the pyactup license):

```
license: https://bitbucket.org/dfmorrison/pyactup/src/master/LICENSE
```

Reply:
done